oceannavigator/frontend/src/components/Map.jsx

```diff
@@ -503,7 +528,117 @@ export default class Map extends React.PureComponent {
           }
 
         }.bind(this),
-      });
+    });
+
+    const anchor = [0.5, 0.5];
+    this.layer_quiver = new ollayer.Vector(
+      {
+        source: null, // set source during update function below
+        style: function(feature, resolution) {
+          const velocity = parseFloat(feature.get("data"));
+          const rotationRads = deg2rad(parseFloat(feature.get("bearing")));
+          if (velocity < knotsToMetersPerSecond(0.5)) {
```

Comment:
These velocity thresholds are defined in the linked S111 standard pdf on page 24